src/views/v2/components/icons/virtual.php

```diff
@@ -19,5 +19,15 @@
 if ( ! empty( $classes ) ) {
 	$svg_classes = array_merge( $svg_classes, $classes );
 }
+
+if ( empty( $icon_title ) ) {
+	$icon_title = __( 'Virtual Event', 'tribe-common' );
```

Comment:
Should this just be `Virtual`? We have replacement words for `Event` if users use a different word.

Reply:
probably - but not in common?
These are meant to be fallbacks while we update all the calls to them across plugins.

Reply:
We _could_ instead have them all default to _no_ label
TBH I am unsure which is the better approach for a few of these (most will fall back to no label, but the three in this PR are used extensively)

Reply:
I know, I think keeping common generic would be better. We don't use `__( 'Featured Event', 'tribe-common' )` or `Recurring` on the other icons.

Reply:
How about we invert it to empty by default, and in the PR for VE I'll change all the calls to it to pass the label?

More work, but best for users?

Reply:
I like your approach Stephen, but leaving it a generic name like `Featured` instead of empty is also ok I think.